libs/tests/userReplicaSetManagerTest.js

```diff
@@ -145,4 +145,9 @@ describe('UserReplicaSetManager Tests', () => {
     assert(secondariesFromChain.length === newSecondaries.length, 'Expect secondary lengths to be equal')
     assert(secondariesFromChain.every((replicaId, i) => replicaId === newSecondaries[i]), 'Secondary mismatch')
   })
+
+  it('Users for creator node', async function () {
+    let resp = await audius1.discoveryProvider.getUsersForCreatorNode(sp1Id)
+    assert(resp.length > 0, 'Expect users to exist for creator node 1')
+  })
```

Comment:
can we not assert on exactly what the users list will be ?

Reply:
added